streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -833,9 +835,15 @@ protected StreamTask createStreamTask(TaskId id, Collection<TopicPartition> part
 
         final ProcessorTopology topology = builder.build(id.topicGroupId);
         final RecordCollector recordCollector = new RecordCollectorImpl(producer, id.toString());
-        return new StreamTask(id, applicationId, partitions, topology, consumer, restoreConsumer, config, streamsMetrics, stateDirectory, cache, time, recordCollector);
+        final long start = System.currentTimeMillis();
```

Comment:
Yeah, my bad. I didn't plan on leaving the debug in there, but i think it is probably useful